app/jobs/audit_manager.rb

```diff
@@ -0,0 +1,14 @@
+class AuditManager
+  def self.system_audit
+    audit = Audit.new(user: User.system_user, packages: Package.stored.count)
```

Comment:
Same as in the controller, just set the user on the audit. Let `start_audit` query and set the count.